### PR TITLE
add hurricane noel example

### PR DESCRIPTION
**Pull request by anamat21**

### CODE REVIEW

File: noel/setrun.py [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+    # == settopo.data values ==
+    topo_data = rundata.topo_data
+    topo_data.topofiles = []
+    # for topography, append lines of the form
+    #   [topotype, minlevel, maxlevel, t1, t2, fname]
+    # See regions for control over these regions, need better bathy data for
+    # the smaller domains
+
+    # use topography data from seperate noel directory
+    topo_path_nc = os.path.join(datadir, 'noel_topo.nc')
```
Comment (mandli) [inline noel/setrun.py:396]: We need to remove the translation of the NetCDF file from the `setrun` file and just refer to the `tt3` file.  Instead we should describe the process of creating the `tt3` file.

File: noel/setrun.py [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+# Time Conversions
+def days2seconds(days):
+    return days * 60.0**2 * 24.0
+
+
+# Scratch directory for storing topo and dtopo files:
+CLAW = os.environ['CLAW']
+
+scratch_dir = os.path.join(os.environ["CLAW"], 'geoclaw', 'scratch')
+datadir = os.path.join(CLAW,'geoclaw','noel') # directory with noel topo data
```
Comment (mandli) [inline noel/setrun.py:34]: For the apps repository we generally do not use the scratch directory but instead either look for a `DATADIR` environment variable or default to the local directory.

File: noel/setplot.py [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+    # Set up for axes in this figure:
+    plotaxes = plotfigure.new_plotaxes()
+    plotaxes.xlimits = [-3, 5]
+    # plotaxes.xlabel = "Days from landfall"
+    # plotaxes.ylabel = "Surface (m)"
+    plotaxes.ylimits = [-2, 3]
+    plotaxes.title = 'Surface'
+
+    # comparison water level files
+    gauges = [np.loadtxt('gauge_data_1.txt'), np.loadtxt('gauge_data_2.txt'),
```
Comment (mandli) [inline noel/setplot.py:178]: If these data files do not exist these plots unfortunately just fail.  We should discuss either putting this in a `try-except` block or automatically downloading them or something else.

**Review by mandli — CHANGES_REQUESTED** [review]
Beyond the comments I also noticed that the following was printed out:
```
*** Computation halted after nv(1) =    12000  steps on coarse grid
GEOCLAW: Frame   32 output files done at time t =  4.75
```
Did you see this as well?

File: noel/setrun.py (hunk 1 above)
Comment (anamat21) [reply noel/setrun.py:396]: Hi should I do the translation in another file then?

File: noel/setplot.py (hunk 3 above)
Comment (anamat21) [reply noel/setplot.py:178]: thanks for the comments, ill look into them! ill also fix the computation halted issue

Comment (anamat21) [reply noel/setplot.py:178]: maybe i can edit comparison_gauge_data.py and then add that code to setplot so it can automatically download the files

**Comment by anamat21** [comment]
Just updated all 5 files! The computation halted error was also fixed! :)

**Pull request merged**